ProjectCoimbra.UWP/Project.Coimbra/Pages/InstrumentsPage.xaml.cs

```diff
@@ -130,7 +137,7 @@ private void ShowPlayersInstrumentInfo()
                         (x.Instrument > -1
                             ? " - " +
                             Instruments[x.Instrument]
-                            : " - Waiting..."))
+                            : $" - {waitingIndicatorResource}"))
```

Comment:
Are you calling this multiple times? If not, you don't need the class variable and you can just get the information within the method.